homeassistant/components/overkiz/sensor.py

```diff
@@ -0,0 +1,418 @@
+        native_unit_of_measurement=TEMP_CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    OverkizSensorDescription(
+        key=OverkizState.CORE_MINIMUM_TEMPERATURE,
+        name="Minimum Temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement=TEMP_CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    OverkizSensorDescription(
+        key=OverkizState.CORE_MAXIMUM_TEMPERATURE,
+        name="Maximum Temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement=TEMP_CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    # AirSensor/COSensor
+    OverkizSensorDescription(
+        key=OverkizState.CORE_CO_CONCENTRATION,
+        name="CO Concentration",
+        device_class=SensorDeviceClass.CO,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    # AirSensor/CO2Sensor
+    OverkizSensorDescription(
+        key=OverkizState.CORE_CO2_CONCENTRATION,
+        name="CO2 Concentration",
+        device_class=SensorDeviceClass.CO2,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    # SunSensor/SunEnergySensor
+    OverkizSensorDescription(
+        key=OverkizState.CORE_SUN_ENERGY,
+        name="Sun Energy",
+        native_value=lambda value: round(float(value), 2),
+        device_class=SensorDeviceClass.ENERGY,
+        icon="mdi:solar-power",
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    # WindSensor/WindSpeedSensor
+    OverkizSensorDescription(
+        key=OverkizState.CORE_WIND_SPEED,
+        name="Wind Speed",
+        native_value=lambda value: round(float(value), 2),
+        icon="mdi:weather-windy",
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    # SmokeSensor/SmokeSensor
+    OverkizSensorDescription(
+        key=OverkizState.IO_SENSOR_ROOM,
+        name="Sensor Room",
+        native_value=lambda value: str(value).capitalize(),
+        entity_registry_enabled_default=False,
+    ),
+    OverkizSensorDescription(
+        key=OverkizState.IO_PRIORITY_LOCK_ORIGINATOR,
+        name="Priority Lock Originator",
+        native_value=lambda value: str(value).capitalize(),
+        icon="mdi:lock",
+        entity_registry_enabled_default=False,
+    ),
+    OverkizSensorDescription(
+        key=OverkizState.CORE_PRIORITY_LOCK_TIMER,
+        name="Priority Lock Timer",
+        icon="mdi:lock-clock",
+        native_unit_of_measurement=TIME_SECONDS,
+        entity_registry_enabled_default=False,
+    ),
+    OverkizSensorDescription(
+        key=OverkizState.CORE_DISCRETE_RSSI_LEVEL,
+        name="Discrete RSSI Level",
+        entity_registry_enabled_default=False,
+        native_value=lambda value: str(value).capitalize(),
+        device_class=SensorDeviceClass.SIGNAL_STRENGTH,
```

Comment:
Missing unit of measurement. When setting a device class for a sensor there must be a unit of measurement according to our dev docs.